packages/beachball/src/bump/setDependentsInBumpInfo.ts

```diff
@@ -19,8 +19,9 @@ export function setDependentsInBumpInfo(bumpInfo: BumpInfo): void {
     const info = packageInfos[pkgName];
     const depTypes = ['dependencies', 'devDependencies', 'peerDependencies'];
     depTypes.forEach(depType => {
-      if (info[depType]) {
-        for (let [dep, _] of Object.entries(info[depType])) {
+      const deps: PackageDeps | undefined = (info as any)[depType];
+      if (deps) {
+        for (let dep of Object.keys(deps)) {
```

Comment:
directly use `for .. in`: `for (let depPackage in deps)`

Reply:
I thought that was still considered bad practice due to `for..in` including non-own properties

Reply:
hm i never heard of it as bad practice. in this use case, i dont think there is a real concern?

Reply:
We have a lint rule against it in Fabric (unless accompanied by a hasOwnProperty check), and my understanding is that it's a common lint rule in other projects too. Object.keys is much more commonly used than for..in across both beachball and oufr.